explorer/src/components/Filters/Filters.tsx

```diff
@@ -132,12 +134,13 @@ export const Filters = () => {
               Clear
             </Button>
           }
-          sx={{ width: 300 }}
+          sx={{ width: 300, alignItems: 'center' }}
         >
-          Filters applied
+          {mixnodes?.data?.length} mix nodes matched your criteria
```

Comment:
can we use `{mixnodes?.data?.length || 0} mix nodes matched your criteria` just in case mixnodes.data is undefined. It's unlikely to be undefined but good to have a fall back